setup.py

```diff
@@ -79,9 +79,11 @@
                       'beautifulsoup4>=4.4.0',
                       'requests>=2.18.1',
                       'lxml>=4.2.1',
+                      'pyarrow>=1.0.1',
                       'pygithub>=1.51',
                       'altair>=4.0.0',
                       'scipy>=1.4.0',
+                      'openpyxl>=3.0.5',
```

Comment:
Why?

Reply:
Snuck in combining the branches. Will fix.